src/mapSearch/mapSearchComponent.tsx

```diff
@@ -287,9 +317,23 @@ const MapSearchComponent = ({
 
                 return (
                   <Fragment key={subtype.id}>
-                    <h3 className="MapSearchComponent__plot-search-subtype-heading">
-                      {subtype.name}
-                    </h3>
+                    <div className="MapSearchComponent__plot-search-subtype-heading">
+                      <h3>{subtype.name}</h3>
+                      <span>
+                        <IconStar size="m" />
+                        {t(
+                          'plotSearchAndCompetitions.mapView.sidebar.legend.star',
+                          '  = add to application'
```

Comment:
These leading spaces don't actually do anything without changing the whitespace CSS rules for the element. In any case I wouldn't bother with doing it that way but by adding a right margin to the icons instead.